codec/decoder/core/src/pic_queue.cpp

```diff
@@ -118,6 +131,21 @@ void FreePicture (PPicture pPic, CMemoryAlign* pMa) {
       pMa->WelsFree (pPic->pBuffer[0], "pPic->pBuffer[0]");
     }
 
+		if (pPic->pMbType) {
```

Comment:
please use astyle to format source code

Reply:
You mean to make the format looking like the following?
How to use "astyle" tool?
Thanks.
void FreePicture (PPicture pPic, CMemoryAlign* pMa) {
  if (NULL != pPic) {
    if (pPic->pBuffer[0]) {
      pMa->WelsFree (pPic->pBuffer[0], "pPic->pBuffer[0]");
      pPic->pBuffer[0] = NULL;
    }

    if (pPic->pMbType) {
      pMa->WelsFree(pPic->pMbType, "pPic->pMbType");
      pPic->pMbType = NULL;
    }
		
    for (int32_t listIdx = LIST_0; listIdx < LIST_A; ++listIdx) {
      if (pPic->pMv[listIdx]) {
        pMa->WelsFree(pPic->pMv[listIdx], "pPic->pMv[]");
        pPic->pMv[listIdx] = NULL;
     }
			
     if (pPic->pRefIndex[listIdx]) {
       pMa->WelsFree(pPic->pRefIndex[listIdx], "pPic->pRefIndex[]");
       pPic->pRefIndex[listIdx] = NULL;
     }
   }
   pMa->WelsFree (pPic, "pPic");
   pPic = NULL;
 }
}

Reply:
code style is defined in build/astyle.cfg, you can type "astyle --options=./build/astyle.cfg *.cpp or *h"  to format the code. thanks!

Reply:
I see. Thx.